website/src/configs/docsList.json

```diff
@@ -94,6 +94,12 @@
     "title": "Svelte / SvelteKit",
     "path": "/docs/guides/sveltejs"
   },
+  {
+    "key": "astro",
+    "title": "Astro",
+    "path": "https://docs.astro.build/en/guides/fonts/#using-fontsource",
+    "isExternal": "true"
```

Comment:
This should be a Boolean value rather than a string.

Reply:
oops